source/pages/albedo.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        df, meta = pvlib.iotools.get_psm3(47.42, -88.05, 'DEMO_KEY', 'assessingsolar@gmail.com',
</pre><p>Would be nice to have each parameter on a single line and add the argument name, e..g,</p><p> </p><p>    df, meta = pvlib.iotools.get_psm3(</p><p>        latitude=47.42,</p><p>        longitude,-88.05,</p><p>        ...</p><p>    )</p><p> </p><p>and so forth</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>